docs/extensions/reference/_posts/1970-01-06-CLI.md

```diff
@@ -327,17 +327,18 @@ By default, `shoutem pack` command will do `npm run build` in both `app` and `se
 
 Run this command from the extension folder.
 
-- ##### shoutem configure [--release]
+- ##### shoutem configure [--release] [--production]
 
 Runs the platform's configure script to sync the local app with native changes to local extensions.
 The flag `--release` is used to switch the local app to the last published configuration on the server.
-Call the `shoutem configure` command again without the flag to switch back to the current development configuration.
+The flag `--production` is used to configure the app's bundle ID for iOS and Android to match the one provided in the Builder. It also activates the code push feature, analytics and push notifications.
```

Comment:
I would add also that that bundle id (iOS) and application id (Android) could be set by changing the values of `"iosBundleId"` and `"androidApplicationId"` in the `config.json`. If they are set, one from the builder would not be respected.